rlkit/core/rl_algorithm.py

```diff
@@ -482,6 +495,128 @@ def get_extra_data_to_save(self, epoch):
             data_to_save['algorithm'] = self
         return data_to_save
 
+
+    ##### Eval stuff #####
+    def obtain_eval_paths(self, deterministic=False):
```

Comment:
would it make it easier to read if this function and `collect_paths` were combined?

Reply:
I think so. I think the only difference is the deterministic thing and the sparsify rewards? The sparse rewards can be handled outside the function by getting it from env infos.

Reply:
done